src/models/models.ts

```diff
@@ -30,27 +28,15 @@ export const setTasks = [
 
 
 
-//"a model should export a function that returns a Sequelize model defintion"
-/*
-export default function (sequelize, Datatypes) {
-    const Category = sequelize.define('Category', {
-        id: DataTypes.UUID,
-        createdAt: DataTypes.DATE,
-        title: DataTypes.STRING,
-        dueDate: DataTypes.DATE,
-        completed: DataTypes.BOOLEAN
-    })
-}
-*/
+
+
 
 /*
 function DB(sequelize, models) {
     this.sequelize = sequelize
     this.models = models
 }
 server.plugins['hapi-sequelizejs'][opts.name] = new DB(opts.sequelize, models);
-
```

Comment:
Big 'ol empty space and commented out code in this file, are these mostly for reference? As long as you prune everything after line 29 before final